Fluid/Ast/IncludeStatement.cs

```diff
@@ -22,13 +22,16 @@ public class IncludeStatement : Statement
         public const string FluidTemplateFactoryKey = "FluidTemplateFactory";
         public const string ViewExtension = ".liquid";
 
-        public IncludeStatement(Expression path)
+        public IncludeStatement(Expression path, Expression value = null)
         {
             Path = path;
+            Value = value;
         }
 
         public Expression Path { get; }
 
+        public Expression Value { get; }
+
         public override async Task<Completion> WriteToAsync(TextWriter writer, TextEncoder encoder, TemplateContext context)
         {
             var relativePath = (await Path.EvaluateAsync(context)).ToStringValue();
```

Comment:
Maybe save the value at this point, and not call `Path.EvaluateAsync` again ?

Reply:
It's similar to `Path`?!!

Reply:
But we need the value of the context!!

Reply:
If I change the code and the tests pass, will you trust me ?

Reply:
Be free to change the code if you have a better solution

> will you trust me ?

I'm always trust you 😄